src/domain/individualCustomer/billsCard/BillsCard.tsx

```diff
@@ -77,22 +78,26 @@ const BillsCard: React.SFC<ComponentProps> = ({
         <LabelValuePair
           align="right"
           label={t('individualCustomer.customerBill.mooring')}
-          value={[mooringFee[0], formatter.format(mooringFee[1])]}
+          value={`${mooringFee[0]}\u00A0\u00A0${formatter.format(
```

Comment:
This is hard to read. Can we move this formatting logic to a dedicated formatting function?